onboard/nav/stateMachine.hpp

```diff
@@ -63,22 +68,22 @@ class StateMachine
 
     NavState executeSearch();
 
-    NavState executeTurnAroundObs();
-
-    NavState executeDriveAroundObs();
-
     void initializeSearch();
 
     bool addFourPointsToSearch();
 
-    Odometry createAvoidancePoint( const double distance );
-
     string stringifyNavState() const;
 
     bool isObstacleDetected() const;
 
     double getOptimalAvoidanceAngle() const;
 
+    double getOptimalAvoidanceDistance() const;
+
+    std::tuple<double, double> getOptimalAvoidanceElements() const;
```

Comment:
Is there a reason to have all `getOptimalAvoidanceAngle`, `getOptimalAvoidanceDistance`, and `getOptimalAvoidanceElements` if they are all private?

Reply:
We need these if an obstacle is seen from the search algorithm. If you prefer, I can make them public, but maybe we should leave that for next year because we really need to start testing this code.